core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -132,7 +132,105 @@ describe('Overview', () => {
             expect(jazzServices_po.getYearVerify().getText()).toEqual('YEAR');
             browser.sleep(5000);
             jazzServices_po.getServiceFromAsset().click();
-      });      
+      });
+
+      it('Verify METRICS Navigation for API' , () => {
+            // Navigation to services
+            browser.wait(EC.visibilityOf(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            //To Navigate to the particular service and verifying the Page
+            jazzServices_po.getAwsServiceName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+            browser.sleep(5000);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+            jazzServices_po.getRefresh().click();
+            browser.sleep(2000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdName()), timeOutHigh);
+            jazzServices_po.getProdName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
+            browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
+            browser.driver.switchTo().activeElement();
+            browser.sleep(5000);
+            jazzServices_po.getMetrices().click();
+            browser.driver.sleep(5000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getDeploymentStatus()), timeOutHigh);
+            jazzServices_po.getDeploymentStatus().click();
+            expect(jazzServices_po.getTestAPI().getText()).toEqual('TEST API');
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getTestAPI()), timeOutHigh);
+            jazzServices_po.getTestAPI().click();
+            browser.sleep(15000);
+            browser.getAllWindowHandles().then(function(handles){
```

Comment:
Just to know, why are we using multiple windows?

Reply:
you must be asking about the getAllWindowHandles() function. It helps to switch between the windows whenever it requires. Like in jazz application when u click in test api button it will automatically open in new window. In that condition we have to handle it using getAllWindowHandles() function.